File: src/main/resources/static/js/filters/statusAgendamento.js
angular.module("societyApp").filter("statusAgendamento", function () {
	return function (input) {

        if(input == "CONFIRMADO"){
            input = "Confirmado";
        } else if (input == "CANCELADO"){
            input = "Cancelado";
        } else if (input == "AGUARDANDO_SOCIETY"){
            input = "Aguardando Society";
        }

		return input;
	};
});

File: agendamento/src/main/java/com/society/gerenciamento/agendamento/domain/enums/StatusAgendamento.java
package com.society.gerenciamento.agendamento.domain.enums;

public enum StatusAgendamento {

	CONFIRMADO, CANCELADO, AGUARDANDO_SOCIETY;

	public static StatusAgendamento getPorString(String confirmacao) {

		if(confirmacao.equalsIgnoreCase(CONFIRMADO.toString())) {
			return CONFIRMADO;
		} else if (confirmacao.equalsIgnoreCase(CANCELADO.toString())) {
			return CANCELADO;
		} else if (confirmacao.equalsIgnoreCase(AGUARDANDO_SOCIETY.toString())) {
			return AGUARDANDO_SOCIETY;
		} else {
			return null;
		}
	}
}
